README.md

```diff
@@ -93,7 +93,7 @@ The `/release` endpoint accepts several parameters:
 * `source_message` git commit message of the new change.
 * `author-email` and `author-name` are base64 encoded http headers. They define the `git author` that pushes to the manifest repository.
 * `version` (optional, but recommended) If not set, Kuberpult will just use `last release number + 1`. It is recommended to set this to a unique number, for example the number of commits in your git main branch. This way, if you have parallel executions of `/release` for the same service, Kuberpult will sort them in the right order.
```

Comment:
please add "fix: " as a prefix for the commit message and PR title

Reply:
Done, thanks!